### ISSUE TITLE
[Keyboard Navigation-Project-My Stuff]Focus does not moves to ‘Pick a name for your project’ after selecting New Project under My Stuff screen.

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2562]
**User Experience:**
Users who are dependent on keyboard navigation or people with mobility impairments who cannot use a mouse will not be able to understand and interact with the page properly if focus does not moves to desired element on the page.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1. Navigate to https://makecode.microbit.org/acc
2. Navigate to the Microbit section element and select Code control given on it.
3. Navigate to the Projects control lying in the header section on the page and select it.
4. Navigate to the MY Stuff tab on the page opened and select New Project option.
5. Navigate to various controls lying on the page opened.
6. Verify that focus moves to the ‘Pick a name for your project’ or not after selecting New Project under My Stuff screen.

**Actual Result:**
Focus does not moves to ‘Pick a name for your project’ after selecting New Project under My Stuff screen.

**Expected Result:**
Focus should moves to ‘Pick a name for your project’ after selecting New Project under My Stuff screen.

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a4bbd37d-ee11-485e-b5c8-0539f0ac94b2}

**Suggested fix:** 
Tips: 
1-All the changes in context should be predictable by user and be triggered by user interaction only 
2-Moving focus should not trigger change in context 
e.g. 
Page should trigger a window/dialog or any expanding menu only when user does the action (SPACEBAR/ENTER) 

**Key References for Fix:** 
https://www.w3.org/TR/WCAG20-TECHS/G107.html  

**Please refer the attachment**
[MAS3.2.1_Project_My Stuff_New Project.zip](https://github.com/Microsoft/pxt/files/1182115/MAS3.2.1_Project_My.Stuff_New.Project.zip)

**Comment by veler** [comment]
@microsoftsam Do you think it is relevant to give the focus to the Project Name field right after creating a project? I am not convinced by that.
But I thought that it could be good to use a aria-live to tell the user that the project has been created.

**Comment by abchatra** [comment]
Closing this by the discussion in the triage team.
